### ISSUE TITLE
Runtime warnings during backend destruction when animations are enabled

### ISSUE THREAD

**Issue by dccsillag** [dccsillag/picom#15]
When shadows and animations are enabled, everything messes up:

- screen gets cleared (with probably uninitialized data)
- nothing is visible
- lots of errors in stdout:

```
[ 26/08/2021 21:49:24.626 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1064)
[ 26/08/2021 21:49:24.626 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1066)
[ 26/08/2021 21:49:24.626 build_shadow ERROR ] Failed to create shadow pixmaps
[ 26/08/2021 21:49:24.629 win_bind_shadow ERROR ] Failed to bind shadow image, shadow will be disabled for 0x01200005 (dzen title)
[ 26/08/2021 21:49:28.331 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1135)
[ 26/08/2021 21:49:28.331 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1137)
[ 26/08/2021 21:49:28.331 build_shadow ERROR ] Failed to create shadow pixmaps
[ 26/08/2021 21:49:28.481 win_bind_shadow ERROR ] Failed to bind shadow image, shadow will be disabled for 0x01e00006 (tap -R src/picom -m release -- --experimental-backends:~/code/picom)
[ 26/08/2021 21:49:28.876 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1148)
[ 26/08/2021 21:49:28.876 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1150)
[ 26/08/2021 21:49:28.876 build_shadow ERROR ] Failed to create shadow pixmaps
[ 26/08/2021 21:49:28.941 win_bind_shadow ERROR ] Failed to bind shadow image, shadow will be disabled for 0x02200006 (ce:~)
[ 26/08/2021 21:49:29.276 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1161)
[ 26/08/2021 21:49:29.276 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1163)
[ 26/08/2021 21:49:29.276 build_shadow ERROR ] Failed to create shadow pixmaps
[ 26/08/2021 21:49:29.336 win_bind_shadow ERROR ] Failed to bind shadow image, shadow will be disabled for 0x01c00645 (picom considerably slower than xcompmgr · Issue #344 · yshui/picom - qutebrowser)
[ 26/08/2021 21:49:30.230 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1174)
[ 26/08/2021 21:49:30.231 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1176)
[ 26/08/2021 21:49:30.231 build_shadow ERROR ] Failed to create shadow pixmaps
[ 26/08/2021 21:49:30.370 win_bind_shadow ERROR ] Failed to bind shadow image, shadow will be disabled for 0x02000003 ((null))
[ 26/08/2021 21:49:31.229 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1187)
[ 26/08/2021 21:49:31.229 x_create_pixmap ERROR ] Failed to create pixmap (X error 11 ALLOC request 53 minor 0 serial 1189)
[ 26/08/2021 21:49:31.229 build_shadow ERROR ] Failed to create shadow pixmaps
[ 26/08/2021 21:49:31.370 win_bind_shadow ERROR ] Failed to bind shadow image, shadow will be disabled for 0x00a00002 (LG3D)
```

**Comment by dccsillag** [comment]
After a reboot, this is no longer the case. I figure it was a one-off thing, where I had previously messed up my Xorg session.

There are still some worrisome warnings that only show up with animations&shadows, though:

```
^C
[ 27/08/2021 21:07:59.428 destroy_backend WARN ] Stale flags set for mapped window 0x02e00006 during backend destruction
[ 27/08/2021 21:07:59.428 destroy_backend WARN ] Stale flags set for mapped window 0x02c0072a during backend destruction
[ 27/08/2021 21:07:59.428 destroy_backend WARN ] Stale flags set for mapped window 0x01000006 during backend destruction
[ 27/08/2021 21:07:59.428 destroy_backend WARN ] Stale flags set for mapped window 0x01200005 during backend destruction
[ 27/08/2021 21:07:59.428 destroy_backend WARN ] Stale flags set for mapped window 0x01600003 during backend destruction
[ 27/08/2021 21:07:59.428 destroy_backend WARN ] Stale flags set for mapped window 0x00a00001 during backend destruction
```

**Comment by dccsillag** [comment]
Actually, it looks like we don't need to have shadows enabled for these warnings to appear -- which makes a lot more sense, given what they are.

**Comment by dccsillag** [comment]
Solved in 3ae7f995bef19c6d5efd236405e28b4c50dd694d.
